app/javascript/application.js

```diff
@@ -2,3 +2,4 @@
 import "@hotwired/turbo-rails"
 import "controllers"
 import "bootstrap"
+import "@fortawesome/fontawesome-free"
```

Comment:
@FaxriddinMaxmadiyorov add an empty line of end file

Reply:
hey, I added an empty line and pushed, could you please check?

Reply:
@XaoGao